app/controllers/application_controller.rb

```diff
@@ -1,44 +1,39 @@
 class ApplicationController < ActionController::Base
-  # Adds a few additional behaviors into the application controller 
+  # Adds a few additional behaviors into the application controller
    include Blacklight::Controller
-  # Please be sure to impelement current_user and user_session. Blacklight depends on 
-  # these methods in order to perform user specific actions. 
+  # Please be sure to impelement current_user and user_session. Blacklight depends on
+  # these methods in order to perform user specific actions.
 
   protect_from_forgery
 
   rescue_from Exception, :with=>:exception_on_website
   layout "frda"
-  
+
   helper_method :show_terms_dialog?, :on_home_page, :on_collection_highlights_page,
                 :on_collections_pages, :on_about_pages, :on_show_page,
                 :on_ap_page, :on_ap_landing_page,
                 :on_images_page, :on_images_landing_page,
                 :on_search_page, :request_path
-  
+
   before_filter :set_locale
-  before_filter :require_http
-  
-  def require_http
```

Comment:
I'd want @jkeck to chime in, in case he remembers why we're doing this.. I'd be concerned about the site possibly loading external resources over http (like, say, the timeline widget), which could break if we're forcing HTTPS.